### PR TITLE
Adding explanation that service containers should not be used in composite actions

### LINKED ISSUE #14905: Update docs about service containers to explain that they should not be used within composite actions
### Code of Conduct

- [X] I have read and agree to the GitHub Docs project's [Code of Conduct](https://github.com/github/docs/blob/main/CODE_OF_CONDUCT.md)

### What article on docs.github.com is affected?

[About service containers](https://docs.github.com/en/actions/using-containerized-services/about-service-containers)

### What part(s) of the article would you like to see updated?

[Introduction part](https://docs.github.com/en/actions/using-containerized-services/about-service-containers#about-service-containers), where it is explained that you can use service containers within your jobs. See [actions/runner issue #1526](https://github.com/actions/runner/issues/1526).

### Additional information

It should be explicitly stated that service containers are meant to be run only for jobs, and not composite actions.

### PR DESCRIPTION
**Pull request by nikola-jokic**
### Why:
Regarding the filed issue #14905, and related to the actions/runner repository [issue #1526](https://github.com/actions/runner/issues/1526), added the explicit line explaining that service containers are not supported to be run inside the composite action

### What's being changed:

Added one line explaining that service containers should not be used in composite actions

### Check off the following:

- [x] I have reviewed my changes in staging (look for "Automatically generated comment" and click **Modified** to view your latest changes).
- [x] For content changes, I have completed the [self-review checklist](https://github.com/github/docs/blob/main/contributing/self-review.md#self-review).

### Writer impact (This section is for GitHub staff members only):

- [ ] This pull request impacts the contribution experience
  - [ ] I have added the 'writer impact' label
  - [ ] I have added a description and/or a video demo of the changes below (e.g. a "before and after video")

This should clarify to users the purpose of the service containers and how should they be used in this version of actions/runner

### CODE REVIEW

**Comment by ramyaparimi** [comment]
@nikola-jokic 
Thanks so much for opening a PR! I'll get this triaged for review :zap:

File: content/actions/using-containerized-services/about-service-containers.md [hunk 1]
```diff
@@ -23,7 +23,7 @@ topics:
 
 Service containers are Docker containers that provide a simple and portable way for you to host services that you might need to test or operate your application in a workflow. For example, your workflow might need to run integration tests that require access to a database and memory cache.
 
-You can configure service containers for each job in a workflow. {% data variables.product.prodname_dotcom %} creates a fresh Docker container for each service configured in the workflow, and destroys the service container when the job completes. Steps in a job can communicate with all service containers that are part of the same job.
+You can configure service containers for each job in a workflow. Service containers are not meant to be used inside composite actions and are not supported to be run inside composite actions. {% data variables.product.prodname_dotcom %} creates a fresh Docker container for each service configured in the workflow, and destroys the service container when the job completes. Steps in a job can communicate with all service containers that are part of the same job.
```
Comment (lucascosti) [inline content/actions/using-containerized-services/about-service-containers.md:26]: Thanks for this update, @nikola-jokic! I'm just going to move the sentence to the end of this paragraph and simplify the wording a little. 

```suggestion
You can configure service containers for each job in a workflow. {% data variables.product.prodname_dotcom %} creates a fresh Docker container for each service configured in the workflow, and destroys the service container when the job completes. Steps in a job can communicate with all service containers that are part of the same job. However, you cannot create and use service containers inside a composite action. 
```

**Review by lucascosti — COMMENTED** [review]
Just one teeny wording change.

**Review by lucascosti — APPROVED** [review]
Ready to merge when the deployment freeze is finished 🚀
